internal/resources/storage.go

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Move this to be a subpackage of tfimport?

Reply:
Just storage.go, or resources?

Reply:
The resources package under internal/tfimport

Reply:
Done.